las_geoh5/export_las.py

```diff
@@ -0,0 +1,205 @@
+#  Copyright (c) 2023 Mira Geoscience Ltd.
+#
+#  This file is part of las-geoh5 project.
+#
+#  All rights reserved.
+#
+
+from __future__ import annotations
+
+from pathlib import Path
+
+from geoh5py.data import ReferencedData
+from geoh5py.objects import Drillhole
+from geoh5py.shared.concatenation import ConcatenatedPropertyGroup
+from lasio import HeaderItem, LASFile
+
+
+def add_well_data(
+    file: LASFile,
+    drillhole: Drillhole,
+) -> LASFile:
+    """
+    Populate las file well data from drillhole.
+
+    :param file: lasio file object.
+    :param drillhole: geoh5py drillhole object.
+
+    :returns: Updated lasio file object.
+    """
+
+    # Add well name
+    file.well["WELL"] = drillhole.name
+
+    # Add epsg code
+    if drillhole.coordinate_reference_system is not None:
+        file.well.append(
+            HeaderItem(
+                mnemonic="GDAT",
+                value=drillhole.coordinate_reference_system["Code"],
+                descr=drillhole.coordinate_reference_system["Name"],
+            )
+        )
+
+    # Add collar data
+    file.well.append(HeaderItem(mnemonic="X", value=float(drillhole.collar["x"])))
+    file.well.append(HeaderItem(mnemonic="Y", value=float(drillhole.collar["y"])))
+    file.well.append(HeaderItem(mnemonic="ELEV", value=float(drillhole.collar["z"])))
+
+    return file
+
+
+def add_curve_data(file: LASFile, drillhole: Drillhole, group):

```

Comment:
Looks like we get duplicate DEPTH TO data

![image](https://github.com/MiraGeoscience/las-geoh5/assets/55204635/057b1b50-3215-4c25-9418-c0994e7937b8)